### PR TITLE
Adapt to CKeyStore.loadKeysOrCertificateChains signature change

### PR DESCRIPTION
**Pull request by jerboaa**
Backport of:
https://github.com/graalvm/mandrel/commit/0c88c7d5ded86bf566ba04660f6588c794131add

JDK 17.0.5 includes the change needed to look for the new signature.

Closes: #428

### CODE REVIEW

**Review by zakkak — APPROVED** [review]

**Comment by jerboaa** [comment]
Thanks, I'll merge this once the merge #429 and the version bump from https://github.com/jerboaa/graal/commit/72173002e20d9770379186446a2ddfdf25e3e71a is in.

**Pull request merged**
